p2p/src/main/java/bisq/network/p2p/network/Connection.java

```diff
@@ -704,9 +704,10 @@ public void run() {
                     long now = System.currentTimeMillis();
                     long elapsed = now - lastReadTimeStamp;
                     if (elapsed < 10) {
-                        log.debug("We got 2 network_messages received in less than 10 ms. We set the thread to sleep " +
-                                        "for 20 ms to avoid getting flooded by our peer. lastReadTimeStamp={}, now={}, elapsed={}",
-                                lastReadTimeStamp, now, elapsed);
+                        if (log.isDebugEnabled())
```

Comment:
I have been in the believe that this is exactly what `log.debug("{}", a)` does before the string is assembled?

Reply:
I think as well that the expression is only executed if log level matches. the static string does not has much cost